WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -12,6 +12,9 @@
 import org.wordpress.android.widgets.WPTextView;
 
 public class DialogUtils {
+
+    private static OpenSansEditText editText;
```

Comment:
Can we avoid `static` field here, and use a Callback that updates the variable in the activity instead?